src/agent/onefuzz-agent/src/main.rs

```diff
@@ -191,20 +191,20 @@ fn run(opt: RunOpt) -> Result<()> {
         }
     }
 
-    telemetry::try_flush_and_close();
+    rt.block_on(telemetry::try_flush_and_close());
```

Comment:
Reuse of the runtime seems reasonable, but I just want to call out [this bit of the docs](https://docs.rs/tokio/latest/tokio/runtime/struct.Runtime.html#multi-thread-scheduler):

> Any spawned tasks will continue running after block_on returns.

Should we maybe play it safe and call [`Runtime::shutdown_background()`](https://docs.rs/tokio/latest/tokio/runtime/struct.Runtime.html#method.shutdown_background) after our main call to `block_on()`?

Reply:
The telemetry data is sent on a background task, so we want it to continue running up until we flush the telemetry client.

Reply:
Ah, good point!